druid-shell/src/gtk/menu.rs

```diff
@@ -44,15 +45,30 @@ impl Menu {
         Menu { items: Vec::new() }
     }
 
-    pub fn add_dropdown(&mut self, menu: Menu, text: &str) {
+    pub fn new_for_popup() -> Menu {
+        Menu { items: Vec::new() }
+    }
+
+    pub fn add_dropdown(&mut self, menu: Menu, text: &str, _enabled: bool) {
+        // TODO: use enabled
         self.items
             .push(MenuItem::SubMenu(strip_access_key(text), menu));
     }
 
-    pub fn add_item(&mut self, id: u32, text: &str, key: impl Into<MenuKey>) {
+    pub fn add_item(
+        &mut self,
+        id: u32,
+        text: &str,
+        key: Option<&HotKey>,
+        _enabled: bool,
+        _selected: bool,
```

Comment:
it would be nice if these worked; I'm okay to merge this as-is but maybe add a `TODO`? (I see you have that above).

In general it would be nice if we had a list of things that aren't implemented, it will get easy to lose track later.